### ISSUE TITLE
Additional padding above buttons on analysis screen 

### ISSUE THREAD

**Issue by bt-abba** [veloce/lichobile#1927]
Issue: 
When analyzing my games, I often click just above the "next move" button, which selects a move further down the list and takes me to that point in the game. 

Proposed solution: 
Either a small amount of additional padding above the game controls for the analysis board, or just slightly larger buttons in that menu in general. This solution seems reasonable given the fact that the only thing which would be cut off by additional padding (or slightly larger menu/buttons) is a line of two of moves.

Alternatives I've considered:
Not fat-fingering the buttons. Unfortunately this approach has failed.

Additional context:
Please see the attached photo (featuring big red circle) for a reference:

![Screenshot_20211125-143316~2](https://user-images.githubusercontent.com/72637132/143497871-fec6ad2f-b082-4233-b4c9-a8bf273f24a6.png)

**Comment by Baikalcadillac** [comment]
I agree, long list of moves are unnecessary and bottom line can be more wide.
Here's my examples: as on website and adapted for the app (sorry for my bad photoshop skill
![aswebsite](https://user-images.githubusercontent.com/67504540/143596931-b5c68b19-1b7b-4f96-9a5d-0ca6f836bad8.jpg)
![mixed](https://user-images.githubusercontent.com/67504540/143596942-bc198f01-4835-4409-9d20-1e2b62ed3291.jpg)
)

**Comment by fbmon** [comment]
Also interested in this fix, you can't scroll back and forth on mobile without accidentally clicking moves from the move list. The bottom menu bar just needs to be a little larger.
